lib/node/index.js

```diff
@@ -296,6 +299,63 @@ Request.prototype.unset = function(field){
 };
 
 /**
+ * Set a filter for accepted status codes (that don't result in an error)
+ *
+ * Examples:
+ *
+ *      req.get('/')
+ *        .set('Accept', 'application/json')
+ *        .set('X-API-Key', 'foobar')
+ *        .acceptedStatus(['2xx', '4xx', 500])
+ *        .end(callback);
+ *
+ *      req.get('/')
+ *        .set({ Accept: 'application/json', 'X-API-Key': 'foobar' })
+ *        .acceptedStatus(function(status) { return status === 200; })
+ *        .end(callback);
+ *
+ * @param {Array|Function} field
+ * @return {Request} for chaining
+ * @api public
+ */
+
+Request.prototype.acceptedStatus = function(filter) {
```

Comment:
If the code for the two is the same, can we start to do some more code sharing? I know we don't do that now but this could be a nice time to re-use common code. Not a blocker tho.